api/application.ts

```diff
@@ -34,12 +36,14 @@ app.use(
         cookie: {
             httpOnly: true,
             maxAge: 1800000,
-            secure: config.secureCookie !== false,
+            // TODO: Possible to deprecate?
+            // secure: config.secureCookie !== false,
+            secure: false,
```

Comment:
We shouldn't deprecate this, in fact if anything we need to enable by default and just disable in lower environments (e.g. preview etc)

Reply:
This a done now. please have a look @paddybasi